### ISSUE TITLE
Configure Modeshape for Avalon Best Practices

### ISSUE THREAD

**Issue by jasoncorum** [WGBH-MLA/avalon-at-wgbh#11]
### Steps / Qustions ticket #11 Enable DB-backed Fedora

1. Database requirements
   1. Automatic backups
   1. Restoration
   1. Minimal maintenance (i.e. no vacuuming)
   1. One very, very large table (10e7)
   1. Sustained load (i.e. bulk ingest)
      1. i.e. don't block with garbage collection, autovacuum, or whatever
   1. Console access
   1. We can restart it
   1. access to logs
   1. Can adjust config options, i.e. timeouts

1. Setting up a database
   1. **???** - Should we...
      * set up our own in RDS?
      * use deployment YAML files in Rancher?
        1. **???** - How are volumes handled?
        1. **???** - How does an arbitrary number of pods communicate with a single DB (or DB cluster). Our use case is batch ingest. Will spinning up multiple pods screw that up, or require special config of DB?
        1. How are backups/recovery handled?
        1. Console access?
        1. Restarting the daemon if needed?
        1. Viewing logs?
        1. Adjusting options?

1. Configuring Fedora
   1. Change this chunk...
      ```
      spec:
        containers:
        - env:
          - name: JAVA_OPTIONS
            value: -Dfcrepo.modeshape.configuration=classpath:/config/file-simple/repository.json
             -Dfcrepo.home=/data
      ```
      Basically change the `file-simple` part of the path to `jdbc-mysql` or `jdbc-postgres`.

  1. Then add other necessary DB connection options (see comment below).

1. **???** - How do then get all of this stuff deployed?

**Comment by jasoncorum** [comment]
Fedora options we'll likely need to change...
```
fcrepo.home=/mnt/fedora-data
fcrepo.mysql.host=db_hostname
fcrepo.mysql.username=db_user
fcrepo.mysql.password=db_password
fcrepo.modeshape.configuration=file:/path/to/fcrepo_code/fcrepo-webapp-4.7.5/WEB-INF/classes/config/jdbc-mysql/repository.json
```

**Comment by afred** [comment]
Complete!

[x] changed config
[x] verified ability to create Collections and Media Objects
[x] verified that Postgres DB is getting populated with data to the tune of 20 records per Collection, and 14 records per Media Object created.
